code/modules/admin/topic.dm

```diff
@@ -4760,7 +4760,20 @@
 				if (alert("Do you wish to activate this religion? You will have to pick a player as its guide. Make sure the player is aware your plans!", "Activating a religion", "Yes", "No") != "Yes")
 					return FALSE
 
-				var/mob/living/carbon/human/preacher = input(usr, "Who should be the leader of this new religion?", "Activating a religion") as mob in world // THERE HAS TO BE A BETTER WAY TO DO THAT
+				var/list/mob/moblist = clients
```

Comment:
I'm fairly sure you want to do `clients.Copy()` here, otherwise you'll be removing clients from the actual `clients` global list.
That is, if the loop below works at all. You're filtering for `mob`s while iterating but `clients` contains, well, `client`s.

Reply:
It worked on my machine
I'll fix that